project.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
No `local-publish` like in the corelibrary? :thinking:

Reply:
I forgot about that, I will add it shortly